python/lsst/sims/utils/stellarMags.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+
+__all__ = ['stellarMags']
+
+
+def calcWDColors():
+    """
+    Calculate a few example WD colors. Values to go in stellarMags(). Here in case
+    values need to be regenerated (different stars, bandpasses change, etc.)
+    """
+
+    from lsst.utils import getPackageDir
+    import os
+    from lsst.sims.photUtils import Bandpass, Sed
+
+    names = ['WD_25000_80', 'HeWD_11000_80', 'WD_3050_80']
+    fns = ['bergeron_25000_80.dat_25000.gz', 'bergeron_He_11000_80.dat_11000.gz', 
+             'bergeron_3000_80.dat_3050.gz']
+    wdDir = os.path.join(getPackageDir('sims_sed_library'), 'starSED/wDs/')
+    files = [os.path.join(wdDir, filename) for filename in fns]
+
+    # Read in the LSST bandpasses
+    bpNames = ['u', 'g', 'r', 'i', 'z', 'y']
+    bps = []
+    throughPath = os.path.join(getPackageDir('throughputs'), 'baseline')
+    for key in bpNames:
+        bp = np.loadtxt(os.path.join(throughPath, 'filter_'+key+'.dat'),
+                        dtype=zip(['wave', 'trans'], [float]*2))
+        tempB = Bandpass()
+        tempB.setBandpass(bp['wave'], bp['trans'])
+        bps.append(tempB)
+
+    # Read in the SEDs and compute mags
+    mags = []
+    for filename in files:
+        star = Sed()
```

Comment:
Sorry.  I see the import statement.

Shouldn't we just make this script a part of sims_photUtils, anyway?

If the answer is "no", can you put the sims_photUtils import statement into a try/except block, again explaining to the user what went wrong.